<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_problems</class>
 <widget class="QWidget" name="test_problems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>391</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="submit_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交/Submit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="problem_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>题目列表</string>
   </property>
   <widget class="QRadioButton" name="problem_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题6</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题7</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题8</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="problem_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>题9</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exit_exam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出/Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_remain">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>建议用时剩余：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="judging_result">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>提交状态</string>
   </property>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="accepted_num">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前通过题数：0</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="problem_details">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
